js/researches/english/EnglishParticiple.js

```diff
@@ -0,0 +1,158 @@
+ *
+ * @returns {boolean} Returns true if the participle is preceded by a given word, otherwise returns false.
+ */
+var includesIndex = function( precedingWords, participleIndex ) {
+	if ( isEmpty( precedingWords ) ) {
+		return false;
+	}
+
+	var precedingWordsEndIndices = [];
+	forEach( precedingWords, function( precedingWord ) {
+		var precedingWordsEndIndex = precedingWord.index + precedingWord.match.length;
+		precedingWordsEndIndices.push( precedingWordsEndIndex );
+	} );
+
+	return includes( precedingWordsEndIndices, participleIndex );
+};
+
+/**
+ * Looks for a specific participle and checks whether if it is directly preceded by a determiner.
+ * If the participle is directly preceded by a determiner it is not a verb, hence the sentence part is not passive.
+ *
+ * @param {string} participle The participle.
+ * @param {string} word The word to check whether it is preceded by a determiner.
+ * @param {string} sentencePart The sentence part to find the participle and word index in.
+ *
+ * @returns {boolean} Returns true if the participle is preceded by a determiner, otherwise returns false.
+ */
+var isPrecededByDeterminer = function( participle, word, sentencePart ) {
+	if ( participle !== word ) {
+		return false;
+	}
+	var indexOfFit = sentencePart.indexOf( word );
+	var determinerMatches = getWordIndices( sentencePart, determinersRegex );
+	return includesIndex( determinerMatches, indexOfFit );
+};
+
+/**
+ * Creates an Participle object for the English language.
+ *
+ * @param {string} participle The participle.
+ * @param {string} sentencePart The sentence part that contains the participle.
+ * @param {object} attributes  The attributes object.
+ *
+ * @constructor
+ */
+var EnglishParticiple = function( participle, sentencePart, attributes ) {
+	Participle.call( this, participle, sentencePart, attributes );
+	this.checkException();
+};
+
+require( "util" ).inherits( EnglishParticiple, Participle );
+
+/**
+ * Sets sentence part passiveness to passive if there is no exception.
+ * @returns {void}
+ */
+EnglishParticiple.prototype.checkException = function() {
+	if ( isEmpty( this.getParticiple() ) ) {
+		this.setSentencePartPassiveness( false );
+		return;
+	}
+	var isPassive =
+		! this.isNonVerbEndingEd() &&
+		! this.hasRidException() &&
+		! this.hasHavingException() &&
+		! this.hasLeftException() &&
+		! this.hasFitException();
+	this.setSentencePartPassiveness( isPassive );
+};
+
+/**
+ * Checks whether a found participle is in the nonVerbsEndingEd list.
+ * If a word is in the nonVerbsEndingEd list, it isn't a participle.
+ * Irregular participles do not end in -ed, and therefore cannot be in the nonVerbsEndingEd list.
+ *
+ * @returns {boolean} Returns true if it is in the nonVerbsEndingEd list, otherwise returns false.
+ */
+EnglishParticiple.prototype.isNonVerbEndingEd = function() {
+	if ( this.getType() === "irregular" ) {
+		return false;
+	}
+	var participle = this.getParticiple();
```

Comment:
I think it is unnecessary to store this.getParticiple() in a participle variable if you're only using it once.